packages/boxel-ui/addon/src/components/icon-button/index.gts

```diff
@@ -31,6 +31,10 @@ class IconButton extends Component<Signature> {
     this.isHoverOnButton = false;
   };
 
+  get hasCustomSizeAttributes() {
+    return this.args.width || this.args.height;
```

Comment:
let's not use falsy here--`width===0` and `height===0` are legit custom values.

```suggestion
  return this.args.width != null || this.args.height != null;
```

Reply:
Are they really? What does passing zero for width or height mean for an icon button?

Reply:
I agree. Perhaps we should be validating that height/width are greater than zero, if present.